app/views/transferAgreementHeader.scala.html

```diff
@@ -0,0 +1,10 @@
+@* helper package contains inputRadioGroup therefore exclude from imports to prevent ambiguous import with view of the same name *@
```

Comment:
Does Play support you putting the partials into a subfolder? This views folder has got quite long, so it could do with some structure. We could start with this change.